src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesVault.md

```diff
@@ -19,34 +20,39 @@ Get-AzRecoveryServicesVault [-ResourceGroupName <String>] [-Name <String>]
 ```
 
 ## DESCRIPTION
+
 The **Get-AzRecoveryServicesVault** cmdlet gets a list of Recovery Services vaults in the current subscription.
 
 ## EXAMPLES
 
 ### Example 1
-```
-PS C:\> Get-AzRecoveryServicesVault
+
+```powershell
+Get-AzRecoveryServicesVault
```

Comment:
examples should have the powershell prompt `PS C:\>`

Reply:
powershell prompt **PS C:\>** was added back.